pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -33,6 +33,7 @@ import (
 	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
 	schedulernodeinfo "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
 	schedulertesting "k8s.io/kubernetes/pkg/scheduler/testing"
+	u "k8s.io/kubernetes/pkg/scheduler/util"
```

Comment:
This is a leftover from a missing rebase, right?

Reply:
Yes.

Reply:
Yes.